### PR TITLE
Sub resources co occurrences asterplot

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

File: public/routines/sub-resources-asterplot.ini [hunk 1]
```diff
@@ -85,21 +83,21 @@ value = env('host')
...
 path = target_title
 value = get(self.titleFieldName).toString()
 
 path = target
-value = pick(['host', 'uri']).values().join('/')
+value = fix(self.uri)
 
 path = weight
-value = fix(self.count / self.total)
+value = fix(self.count / self.hitsTotal)
```
Comment (jj618) [inline public/routines/sub-resources-asterplot.ini:100]: If hitsTotal = 0 then divid by 0

**Review by jj618 — CHANGES_REQUESTED** [review]
Can you try to fix this div by 0, please.

**Comment by AlasDiablo** [comment]
commit 71cde1b have a big side effect...

**Review by jj618 — APPROVED** [review]
LGTM

**Pull request merged**